src/main/java/Gops/TaskList.java

```diff
@@ -0,0 +1,203 @@
+     */
+    public void addNewEvent(String eventDescription, String startDate, String endDate, boolean status) {
+        Event event = new Event(eventDescription, startDate, endDate);
+        event.todoStatus = status;
+        taskList.add(event);
+    }
+
+    /**
+     * Adds new event object to task list
+     * @param eventDescription task description
+     * @param startDate task start date
+     * @param endDate task end date
+     */
+    public void addNewEvent(String eventDescription, String startDate, String endDate) {
+        Event event = new Event(eventDescription, startDate, endDate);
+        taskList.add(event);
+    }
+
+    /**
+     * Removes specific task from list
+     * @param index task index
+     */
+    public void removeTask(int index) {
+        taskList.remove(index);
+    }
+
+    /**
+     * Gets specific task from list
+     * @param index task index
+     * @return task
+     */
+    public Todo getTask(int index) {
+        return taskList.get(index);
+    }
+
+    /**
+     * Returns size of task list
+     * @return task list size
+     */
+    public int size() {
+        return taskList.size();
+    }
+
+    /**
+     * Prints the task at the end of the list
+     */
+    public String printNewestTask() {
+        StringBuilder sb = new StringBuilder();
+        sb.append("Got it. I've added this task:\n");
+        sb.append("------------------------------------------------------------------\n");
+        sb.append(taskList.get(taskList.size() - 1).stringPrinter());
+        sb.append("\n------------------------------------------------------------------\n");
+        sb.append("Now you have ").append(taskList.size()).append(" tasks in the list.");
+        String confirmationMessage = sb.toString();
+        return confirmationMessage;
+
+    }
+
+    /**
+     * Prints task at specified index
+     * @param index task index
+     */
+    public void printTask(int index) {
+    taskList.get(index).Printer();
+    }
+
+    /**
+     * Returns string representation of specific task
+     * @param index task index
+     * @return string rep of task
+     */
+    public String stringPrintTask(int index) {
+        return taskList.get(index).stringPrinter();
+    }
+
+    /**
+     * Changes status of task at specified index
+     * @param index task index
+     * @param status task status
+     */
+    public void changeTaskStatus(int index, boolean status) {
+        taskList.get(index).todoStatus = status;
+    }
+
+    /**
+     * Returns tasklist status
+     * @return task list status
+     */
+    public boolean isEmpty() {
+        return taskList.isEmpty();
+    }
+
+    /**
+     * Prints string representation of task list
+     */
+    public String listPrinter() {
+        StringBuilder stringBuilder = new StringBuilder();
+        stringBuilder.append("Here are the tasks in your list\n");
+        stringBuilder.append("------------------------------------------------------------------\n");
```

Comment:
Perhaps this can be replaced with e.g. `"-".repeat(50)` for readability. Could be it is stored as a constant since it is repeated elsewhere.